src/instructlab/system/system.py

```diff
@@ -0,0 +1,22 @@
+# Standard
+import logging
+
+# Third Party
+from click_didyoumean import DYMGroup
+import click
+
+# Local
+from .reset import reset
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(cls=DYMGroup)
+@click.pass_context
+def system(ctx):
+    """Command group for all system-related command calls"""
+    ctx.obj = ctx.parent.obj
+    ctx.default_map = ctx.parent.default_map
+
+
+system.add_command(reset)
```

Comment:
Could you please move `sysinfo` here, too?

Reply:
Sure thing!

Reply:
Done.